aries_vcx/tests/test_pool.rs

```diff
@@ -87,4 +88,19 @@ mod integration_tests {
         assert_eq!(expect_service, service)
         }).await;
     }
+
+    #[tokio::test]
+    async fn test_add_get_service_public() {
+        SetupWalletPool::run(|setup| async move {
+
+            let did =setup.institution_did.clone(); //"7FfoFcg3MisAPRrxjHFr5P".to_string(); //
+            let create_service = EndpointService::default();
```

Comment:
let's call this var `endpoint_didsov`. Also let's actually set up some routing keys on it explicitly. The test silently assumes the default() implementation for both EndpointService and AriesService, I'd prefer setting and asserting values more explicitly

Reply:
Could you give an example?

Reply:
Here's my suggestion for this test:
```
    #[tokio::test]
    async fn test_add_get_service_public() {
        SetupWalletPool::run(|setup| async move {
            let did = setup.institution_did.clone();
            let create_service = EndpointDidSov::create()
                .set_service_endpoint("https://example.org".into())
                .set_routing_keys(Some(vec!["did:sov:456".into()]));

            write_endpoint(setup.wallet_handle, setup.pool_handle, &did, &create_service).await.unwrap();
            thread::sleep(Duration::from_millis(50));
            let service = get_service(setup.pool_handle, &Did::new(&did).unwrap()).await.unwrap();

            let expect_recipient_key = get_verkey_from_ledger(setup.pool_handle, &setup.institution_did).await.unwrap();
            let expect_service = AriesService::default()
                .set_service_endpoint("https://example.org".into())
                .set_recipient_keys(vec![expect_recipient_key])
                .set_routing_keys(vec!["did:sov:456".into()]);
            assert_eq!(expect_service, service)
        }).await;
    }
    ```
I see you just followed example by the original test `test_add_get_service` - since we are at it, could you update that test in similar fashion?